src/Controller/InstallProjectController.php

```diff
@@ -110,6 +110,10 @@ public function configureAction(Request $request)
 
         if ($inputData->has('configuration/php_cli')) {
             $tensideConfig->set('php_cli', $inputData->get('configuration/php_cli'));
+        } elseif ('' !== PHP_BINARY && file_exists(PHP_BINARY)) {
```

Comment:
Shouldn't we maybe call the `autoconfig` route first? Is that the purpose of the `autoconfig` route?